### ISSUE TITLE
[CoE Starter Kit - BUG] Admin View Premium API Column not accurate

### ISSUE THREAD

**Issue by nasilvae** [microsoft/coe-starter-kit#5743]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Hi Team,

When you expand the premium api column under the apps table in the admin which provides a true/false statement is not complement and is missing information.

### Expected Behavior

Premium API column under the Apps table should be fully populated.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

May 2023 with hotfix

### What app or flow are you having the issue with?

Admin View

### What method are you using to get inventory and telemetry?

Data Export

### Steps To Reproduce

Open Admin View
Add Premium API Column.

### Anything else?

https://github.com/microsoft/coe-starter-kit/issues/5640

**Comment by Jenefer-Monroe** [comment]
is this just something you found while testing or is this a field you are actively using?

**Comment by nasilvae** [comment]
> is this just something you found while testing or is this a field you are actively using?

both.

**Comment by Jenefer-Monroe** [comment]
ok no change has happened here so it could be a change in what is brought down. In the meantime until we can investigate you can use the Power BI where you can look for the premium marker.

**Comment by manuelap-msft** [comment]
I think this is something we just consume directly from the Power Apps for Admins connector - Get Apps as Admin action. It's not something that's calculated in the kit.

If you look at the output from Get Apps as Admin, you'll find this section in the properties:
                    "appPlanClassification": "Premium",
                    **"usesPremiumApi": true,**
                    "usesOnlyGrandfatheredPremiumApis": false,
                    "usesCustomApi": false,
                    "usesOnPremiseGateway": false,
                    "usesPcfExternalServiceUsage": false,
                    "isCustomizable": true,

Can you check the output from Get Apps as Admin directly in a separate flow, is the value correct there and incorrect in the CoE kit for the same app? Or incorrect in both?

**Comment by Jenefer-Monroe** [comment]
We've seen this issue before for caches apps/flows. Where this call would return something but its something that cant be parsed like that.

**Comment by nasilvae** [comment]
> We've seen this issue before for caches apps/flows. Where this call would return something but its something that cant be parsed like that.

its interesting right. What do you recommend to test more.

**Comment by Jenefer-Monroe** [comment]
I dont think this is related to the issue of the premium api issue. I've sent people with this issue around the not-json to product support in the past to get in a clean state. We;ve not been able to do anything to resolve ourselves unfortunately.

**Comment by Jenefer-Monroe** [comment]
I do not repro this. I do not have any blanks for this field in a June install, and spot checked it against an old build (Oct 2022) and the values seem to be the same.
